Sources/StreamChatUI/Utils/TextViewMentionedUsersHandler.swift

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// The component responsible to get the tapped mentioned user in a `UITextView`.
+open class TextViewMentionedUsersHandler {
+    /// Get the tapped mentioned user in a text view.
+    /// - Parameters:
+    ///   - textView: The `UITextView` being tapped.
+    ///   - characterRange: The location where the tap was performed.
+    ///   - mentionedUsers: The current mentioned users in the message.
+    /// - Returns: The `ChatUser` in case it tapped a mentioned user.
+    open func mentionedUserTapped(
+        on textView: UITextView,
+        in characterRange: NSRange,
+        withMentionedUsers mentionedUsers: Set<ChatUser>
+    ) -> ChatUser? {
+        guard let text = textView.text,
+              let range = Range(characterRange, in: text)
+        else {
```

Comment:
Weird formatting

Reply:
It looks fine to me, if we want to be more strict here, we should introduce rules in SwiftFormat, to avoid wasting time with formatting comments on PR Reviews